src/php/ext/grpc/call.c

```diff
@@ -244,6 +244,11 @@ PHP_METHOD(Call, __construct) {
   gpr_mu_unlock(&channel->wrapper->mu);
 }
 
+char* grpc_code_to_string($code) {
+  // TODO: swtich $code and return related string.
```

Comment:
We should do this now?

Reply:
I am sorry. This is a WIP describing what my idea is. I haven't implement it yet because I am not sure should I do this in C layer or in PHP layer. And also should this just be a helper function or add a field to the `$status`.

I will also add tests after the decision is made.

Reply:
I see. Doing this in the PHP layer may be good enough for now. So the helper.php below should suffice.